src/components/expanded-state/chart/chart-data-labels/ChartChangeDirectionArrow.js

```diff
@@ -11,26 +12,29 @@ const ArrowIcon = styled(Icon).attrs({
   name: 'fatArrow',
 })``;
 
-export default function ChartChangeDirectionArrow({ ratio }) {
+export default function ChartChangeDirectionArrow({ ratio, sharedRatio }) {
   const { colors } = useTheme();
+  const { isActive } = useChartData();
```

Comment:
same here via `ChartExpandedStateHeader` -> `ChartPercentChangeLabel`

Could we just pass along `isActive` from there as prop so that it doesn't have to run all the effects and calculations within `useChartData` again

Reply:
I guess you were looking at a wrong hook.

This hook only reads the value from the context:
https://github.com/rainbow-me/rainbow/blob/develop/src/react-native-animated-charts/src/helpers/useChartData.ts